front/pages/api/registry/[type]/lookup.ts

```diff
@@ -230,7 +215,6 @@ async function handleDataSourceView(
 
 async function handleDataSource(
   auth: Authenticator,
-  groups: GroupResource[],
   dataSourceId: string
 ): Promise<Result<LookupDataSourceResponseBody, Error>> {
```

Comment:
Let's add a log here to track usage of dataSource (irrepsective of id/name)

Reply:
Done.